packages/PoppyPrint-Core.package/PPFormatter.class/class/formatString.class.noPattern.notifying.with..st

```diff
@@ -15,6 +15,8 @@ formatString: aString class: aClass noPattern: aBoolean notifying: anObject with
 			notifying: anObject
 			ifFail: nil.
 	
+	(methodNode hasPragma: #ppIgnore) ifTrue: [^ aString].
```

Comment:
consistent naming convention from issue #52 and tests would be `#PPIgnore`

Reply:
Ich hatte nochmal mit Christoph darüber gesprochen und er meinte, dass wir #ppIgnore und nicht #PPIgnore benutzen sollen.

Reply:
achso okay, sowas gerne im issue nochmal vermerken oder kommentieren:)

Reply:
ich hab jetzt das jetzt direk im issue geändert...